portfolio/src/main/java/com/google/sps/servlets/MyServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+// import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/information")
+public final class MyServlet extends HttpServlet {
+
+  String[] info = {"Hi", "This is about", "My information"};
+
+  String[] list;
+
+  public MyServlet(String[] list){
+      this.list = list;
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Convert the server info to JSON
+    MyServlet myservlet = new MyServlet(info);
+    String json = convertToJson(myservlet);
```

Comment:
Minor: This would work. However, it may be easier to follow the code if each class has one purpose. This is sometimes called separation of concerns. Usually, the Servlet class has the purpose of responding to HTTP requests. Each type of data that's exchanged between the Web browser, and the server would have its own (data) class. E.g.:

```Java
class MyInformation {
  public String[] messages;
}
```

Then, your servlet's `doGet` could have code like this:

```Java
MyInformation information = new MyInformation();

information.messages = info;

String json = convertToJson(information);
```

If the result is as simple as a list of strings, you could do without a separate data class, as well. E.g., you could pass `String[]` to `convertToJson`.